gcp/cloudrun/main.tf

```diff
@@ -0,0 +1,42 @@
+variable project_id {} # Required
+
+resource "google_project_service" "project" {
+  project = "${var.project_id}"
+  service = "run.googleapis.com"
+
+  disable_dependent_services = true
+}
+
+resource "google_cloud_run_service" "default" {
+  name     = "tftest-cloudrun"
+  location = "us-central1"
+  provider = "google-beta"
+  project  = "${var.project_id}"
+
+  metadata {
+    namespace = "${var.project_id}"
+  }
+
+  spec {
+    containers {
+      image = "gcr.io/cloudrun/hello"
+    }
+  }
+
+  depends_on = [
```

Comment:
Same here

Reply:
If removed, the creation of this resourse starts at the same time as the api service being enabled, so It fails. Terraform does not see it as a dependency so I had to add it manually